docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,51 @@
+= PostgreSQL by VSHN Availability check
+
+== Problem
+
+We need to monitor availability of PostgreSQL by VSHN, based on:
+
+* SLA: 99.99%
+* SLO: 99.995%
+* SLI: 99.995%
+
+== Requirements for SLI
+
+* PostgreSQL is accessible on network level
+* The end user is able to login and execute database queries
+
+== Solutions
+
+1. Integrated postgresql_exporter into Stackgres cluster
+2. Postgresql_exporter managed and run by VSHN
+3. Customly written exporter
```

Comment:
This is still very barebone. What would a customer exporter include? What would a separate postresql_exporter mean?